docs/releases/changelog.rst

```diff
@@ -7,6 +7,14 @@ Tracpro's version is incremented upon each merge to master according to our
 We recommend reviewing the release notes and code diffs before upgrading
 between versions.
 
+v1.5.3 (released 2017-04-10)
+----------------------------
+
+Code diff: https://github.com/rapidpro/tracpro/compare/v1.5.3...develop
```

Comment:
Should this be 

     https://github.com/rapidpro/tracpro/compare/v1.5.2...v1.5.3   ?